indico/modules/events/timetable/controllers.py

```diff
@@ -109,3 +113,17 @@ def _process_PATCH(self):
         if updates:
             update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+
+class RHTimetable(RHConferenceBaseDisplay):
+    def _checkParams(self, params):
+        RHConferenceBaseDisplay._checkParams(self, params)
+        self.event = self._conf
+        self.layout = request.args.get('ttLyt', '')
```

Comment:
Would it be much work to change this awful name (and just keep it as a fallback in case people link to it)?

    self.layout = request.args.get('layout', request.args.get('ttLyt', ''))